test/chtest-make-auto

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
Script needs a comment explaining what it does and how to use it.

Reply:
The thought here is that this script belongs in libexec and shouldn't be used directly outside of ch-test. I'll go ahead and comment on that.

Should I make a libexec directory now and move ch-test (along with base.sh there)?

Reply:
I'm creating a `libexec` directory in my `autotools2` branch. I'm a little skeptical of creating it in a 2nd branch for fear of merge conflicts. So I'd suggest leaving it where it is for now.

All that said, it still needs a comment. :)

Reply:
> Should I make a libexec directory now and move ch-test (along with base.sh there)?
I am reluctant spending time moving stuff around if autotools is just going to refactor everything anyway.

Reply:
Sorry, to be more clear, no I don't think you should do those moves. I agree with you on the reasoning.